### PR TITLE
Accept SIS IDs for accounts and terms; allow multiple terms

### PR DESCRIPTION
**Pull request by cmurtaugh**
* allow the account-id parameter to be passed as either the numeric Canvas ID or the sis_account_id
* allow the enrollment-term-id parameter to be passed as either the numeric Canvas ID or the sis_term_id
* allow multiple enrollment-term-id values to be passed in

### CODE REVIEW

File: mailing_list/management/commands/activate_all_lists.py [hunk 1]
```diff
@@ -19,21 +19,26 @@ class Command(BaseCommand):
...
-            raise CommandError(str(e))
-
+        courses = []
+        for term_id in options['enrollment_term_id']:
+            try:
+                courses += get_courses_for_account_in_term(
+                    options['account_id'], term_id
+                ) or []
+            except CanvasAPIError as e:
+                # don't raise the error
```
Comment (rascalking) [inline mailing_list/management/commands/activate_all_lists.py]: Maybe log it, at least?

File: mailing_list/management/commands/activate_all_lists.py [hunk 2]
```diff
@@ -19,21 +19,26 @@ class Command(BaseCommand):
     help = 'Activates all mailing lists for a given account and enrollment term'
 
     def add_arguments(self, parser):
-        parser.add_argument('--account-id', type=int, required=True,
-                            help='Canvas account id')
-        parser.add_argument('--enrollment-term-id', type=int, required=True,
-                            help='Canvas enrollment term id')
+        parser.add_argument('--account-id', required=True,
+                            help='Canvas account id (e.g. 29) or SIS account ID (e.g. sis_account_id:school:hls)')
+        parser.add_argument('--enrollment-term-id', nargs='*', required=True,
```
Comment (rascalking) [inline mailing_list/management/commands/activate_all_lists.py]: Just curious...does `nargs='*', required=True` require the option, but allow for no values to it?  ie. lets them explicitly not set it, but won't let you forget to pass it?

File: mailing_list/management/commands/activate_all_lists.py [hunk 3]
```diff
@@ -19,21 +19,26 @@ class Command(BaseCommand):
     help = 'Activates all mailing lists for a given account and enrollment term'
 
     def add_arguments(self, parser):
-        parser.add_argument('--account-id', type=int, required=True,
-                            help='Canvas account id')
-        parser.add_argument('--enrollment-term-id', type=int, required=True,
-                            help='Canvas enrollment term id')
+        parser.add_argument('--account-id', required=True,
+                            help='Canvas account id (e.g. 29) or SIS account ID (e.g. sis_account_id:school:hls)')
```
Comment (rascalking) [inline mailing_list/management/commands/activate_all_lists.py]: Style whining: can we limit this to 80 columns, please?

**Comment by rascalking** [comment]
:+1:

File: mailing_list/management/commands/activate_all_lists.py (hunk 1 above)
Comment (cmurtaugh) [inline mailing_list/management/commands/activate_all_lists.py]: It's actually getting logged upstream, so I didn't log it again here.

File: mailing_list/management/commands/activate_all_lists.py (hunk 2 above)
Comment (cmurtaugh) [inline mailing_list/management/commands/activate_all_lists.py]: Ah - yeah - that'd be better off as nargs='+' here. I'll change it.

**Pull request merged**